XamBasePacket/Converters/IntToBoolConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace XamBasePacket.Converters
+{
+	public class IntToBoolConverter : IValueConverter
```

Comment:
Better to find a better name than generic `IntToBool`. In this particular case we comparing only greater then zero cases, we could add an equal zero and less than zero, or more complex int to bool converters in the future.

Reply:
IntToBool - in the future, it needs full refactor for all other generic types